sdk/harambe/utils.py

```diff
@@ -3,8 +3,22 @@
 from typing import Callable, List, Optional, ParamSpec, TypeVar, Union
 from urllib.parse import urljoin
 
+from nanoid import generate
 from playwright.async_api import Locator, Page
 
+alphabet = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+size = 16
+
+
+def nanoid_factory() -> str:
```

Comment:
Stolen from deworker for unique file names

Reply:
why not just UUID v4 here so we dont need another dep